### DISCUSSION TITLE
Welcome to jimp Discussions!

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [oliver-moran/jimp#1142] [Announcements]
Lets try to keep questions here!

❓ Have a question about jimp or how to use it? This is the place to ask! Try searching first though to see if your question has already been answered

🔧 We also have discussions about plans for changes to jimp here too! Get involved and help shape the direction of the project 🚀 

Want to get involved? Search for issues with the label `solution-in-issue` for easy wins!

More info [here](https://github.com/jimp-dev/jimp/issues/1190).

**Comment by Njinou** [comment]
Hello I am having issues using Jimp on a React Native project version 0.22.0
the is the error am getting .... [TypeError: _$$_REQUIRE(_dependencyMap[9], "jimp").read is not a function. (In '_$$_REQUIRE(_dependencyMap[9], "jimp").read('https://media.geeksforgeeks.org/wp-content/uploads/20190328185307/gfg28.png')', '_$$_REQUIRE(_dependencyMap[9], "jimp").read' is undefined)
I would appreciate any help

**Reply by hipstersmoothie** [reply]
Can you open an issue for this with a reproducible example

**Reply by Njinou** [reply]
Sure
